[ADD] Employees: Offboarding an employee (#11040)
odoo/documentation: content/applications/hr/employees/offboarding.rst

who left the company for the :guilabel:`Last 365 Days`.

To only view the data for the third quarter, click the :icon:`fa-caret-down` :guilabel:`(down
arrow)` in the search bar, revealing multiple filter and grouping options. Click :guilabel:`Last 365
Days` in the :icon:`fa-filter` :guilabel:`Filters` column, to turn off that filter. Then click
:guilabel:`Date`, and click :guilabel:`Q3` from the resulting drop-down menu. Click off of the
drop-down menu to close it.

Next, click :icon:`fa-minus-square-o` :guilabel:`Total` at the top of the rows to collapse it. After
doing so, only one number appears on the pivot table: the total number of employees who left the
company in the third quarter. In this example, there are ten.

To view the various departments where employees left, click :icon:`fa-plus-square` :guilabel:`Total`
at the top of the rows, revealing a drop-down menu, and click :guilabel:`Department`. Click away
from the drop-down to close it, and now the pivot table displays the total number of employees who
left (:guilabel:`# Departure Employee`) during the third quarter, organized by department.

> In this example, it can be determined that out of the tn employees who left, the

Reviewer: I would consider turning this last section into an example admonition block since it is more specific than the rest, which is instructions on how to build the report. Since this image and the "in this example" parts are talking about a specific report, it would fit better in an example, than in general instructions.

Author: OK, after looking at this doc, I decided that this section (the reporting part) should be in its own "Employee Retention Report" doc, and the offboarding only should be in this doc. So- I am moving this to a new doc. I know, crazy! But it seems like it's a better fit (I like to have the reports separate). I needed a little while to make sense of this doc!

===========
Offboarding
===========

When an employee leaves the company, it is important to ensure their employee record is updated to
reflect their departure, log the reason why, close any open activities associated with the
employee, and provide them with any important documents.

Archive an employee
===================

In Odoo, when an employee leaves the company they must be *archived*. To archive an employee, first

> navigate to the main **Employees** app. From here, locate the employee who is leaving the company,

Reviewer 2: ```suggestion
navigate to the :menuselection:`Employees app`. From here, locate the employee who is leaving the company,
```
when instructing the reader to "navigate" somewhere, the menuselection formatting is used 👍

and click on their employee card.

The employee form loads, displaying all their information. Click the :icon:`fa-gear`
:guilabel:`(gear)` icon in the top-left corner, and a drop-down menu appears. Click
:icon:`oi-archive` :guilabel:`Archive`, and an :guilabel:`Employee Termination` pop-up window
appears.

Fill out the following fields on the form:

- :guilabel:`Departure Reason`: Select a reason the employee is leaving from the drop-down menu. The
  default options are:

  - :guilabel:`Fired`: Select this option when an employee is being let go, and the company has
    given notice.
  - :guilabel:`Resigned`: Select this option when the employee no longer wishes to be employed, and
    the employee has given notice.
  - :guilabel:`Retired`: Select this option when the employee is retiring.
  - :guilabel:`Became Freelance`: Select this option when the employee is no longer working for the
    company, but is becoming a freelance worker instead.

- :guilabel:`Contract End Date`: Using the calendar selector, select the last day the employee is
  working for the company.

> - :guilabel:`Detailed Reason`: Enter a short description for the employee's departure.

Reviewer 2: ```suggestion
- :guilabel:`Detailed Reason`: Enter a short description for the employee's departure in this field.
```

- :guilabel:`Close Activities`: Tick the checkbox next to each type of activity to close or delete

>   any open activities associated with it. It is recommended to tick **all** boxes that are

Reviewer 2: ```suggestion
  any open activities associated with it. It is recommended to tick **all** checkboxes that are
```

  applicable. The available options are:

>   - :guilabel:`Appraisals`: Cancels all appraisals scheduled after the contract end date.
>   - :guilabel:`Contract`: Applies an end date for the current contract.
>   - :guilabel:`Company Car`: Removes the employee as the driver for their current company car, and
>     :ref:`assigns the next driver <fleet/new_vehicle/new-driver>`, if applicable.
>   - :guilabel:`Time Off`: Cancels any time off requests after the contract end date.
>   - :guilabel:`Allocations`: Removes the employee from any accrual plans they are on.

Reviewer 2: ```suggestion
  - :guilabel:`Appraisals`: cancels all appraisals scheduled after the contract end date.
  - :guilabel:`Contract`: applies an end date for the current contract.
  - :guilabel:`Company Car`: removes the employee as the driver for their current company car, and
    :ref:`assigns the next driver <fleet/new_vehicle/new-driver>`, if applicable.
  - :guilabel:`Time Off`: cancels any time off requests after the contract end date.
  - :guilabel:`Allocations`: removes the employee from any accrual plans they are on.
```
opting for lowercase in this instance, since the descriptions following the colon aren't complete sentences.

> - :guilabel:`HR Info`: Tick the box next to :guilabel:`Send Access Link` to send a download link to

Reviewer 2: ```suggestion
- :guilabel:`HR Info`: Tick the checkbox next to :guilabel:`Send Access Link` to send a download link to
```

>   the employee's personal email address, containing all their personal HR files they own.

Reviewer 2: ```suggestion
  the employee's personal email address, containing all their personal HR files.
```

- :guilabel:`Private Email`: This field appears if the :guilabel:`HR Info` checkbox is ticked. Enter
  the private email address for the employee.

> When the form is complete, click :guilabel:`Done`. The employee record is archived, an email with a

Reviewer 2: ```suggestion
When the form is complete, click :guilabel:`Apply`. The employee record is archived, an email with a
```
![Screenshot 2024-09-18 093434](https://github.com/user-attachments/assets/3a57873f-6652-451d-a173-5de8ccbc225e)

download link to their personal documents is sent to the employee's private email address (if
selected), and a red :guilabel:`Archived` banner appears in the top-right corner of the employee
form. The chatter logs the :guilabel:`Departure Date` and :guilabel:`Departure Reason`, and if an
access link was emailed.

.. image:: offboarding/termination.png
   :align: center
   :alt: The employee termination form with all fields filled out.

Send HR documents access link
-----------------------------

> If the access link was not sent when first archiving the employee on the :guilabel:`Employee
> Termination` form, it can be sent after the employee is archived at any point.

Reviewer 2: ```suggestion
If the access link was not sent when first archiving the employee on the *Employee Termination*
form, it can be sent after the employee is archived at any point.
```
opting for italics here because the form is no longer present at this moment

After an employee is archived, they are no longer visible on the main **Employees** app dashboard.

> To view the archived employees, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` in the

Reviewer 2: ```suggestion
To view the archived employees, navigate to the :menuselection:`Employees app` dashboard, and click the :icon:`fa-caret-down` :guilabel:`(down arrow)` in the
```

> search bar to reveal a drop-down menu. Click :guilabel:`Archived`, towards the bottom of the

Reviewer 2: ```suggestion
search bar to reveal a drop-down menu. Select :guilabel:`Archived`, towards the bottom of the
```

> :icon:`fa-filter` :guilabel:`Filters` column, then click off the pop-up window to close it.

Reviewer 2: ```suggestion
:icon:`fa-filter` :guilabel:`Filters` column, then click away from the drop-down window to close it.
```

> Now only archived employees appear on the dashboard. Click on the desired employee to open their

Reviewer 2: ```suggestion
Now, only archived employees appear on the dashboard. Click on the desired employee to open their
```

employee form. On this form, click the :icon:`fa-gear` :guilabel:`(gear)` icon in the top-left
corner, then click :guilabel:`Send HR Documents Access Link` from the resulting drop-down menu. The

> chatter logs that the link was sent.

Reviewer 2: When I attempted to do this with a few archived employees, per your instructions, I received the following 'Invalid Operation' pop-up message:
![Screenshot 2024-09-18 094033](https://github.com/user-attachments/assets/5b2f6a88-7d0d-4f8b-b857-21b1e92c6c45)
I'd suggest including reasons why that would occur, as well as some elaboration on how to avoid/fix such an error, should that happen to the reader.
